src/config/dynamic-theme-fixes.config

```diff
@@ -3920,6 +3929,15 @@ body {
 
 ================================
 
+promods.net
+
+CSS
+html {
+    background-image: none !important;
```

Comment:
The background-image is already dark by default not sure why removing a non bright image.

Reply:
Because it messes with contrast settings. The image is originally calibrated to the website's grey background; however, it breaks with increased contrast.
![Without dark reader](https://user-images.githubusercontent.com/5833034/89384084-3c51f080-d706-11ea-951b-08fc1f68cd56.png)
![Dark reader, background image removal, max contrast](https://user-images.githubusercontent.com/5833034/89384144-52f84780-d706-11ea-905b-789b2ab7706c.png)
You see, this fix makes the website *darker*.